lib/config/options/index.ts

```diff
@@ -2042,6 +2042,14 @@ const options: RenovateOptions[] = [
     experimental: true,
     experimentalIssues: [20542],
   },
+  {
+    name: 'containerVulnerabilityAlerts',
```

Comment:
Why is this config option introduced? I'm thinking that when we enable the CVE alerts, we'll have to enable both `osvVulnerabilityAlerts` and `containerVulnerabilityAlerts` at the same time to make them work out of the box, or is the idea that users will be able to turn them on/off according to their needs?

Reply:
That makes a lot of sense, thanks for the explanation.